File: Soumadip/Array/PalindromeArr.java
// program to check array is palindrome or not
package Soumadip.Array;

import java.util.*;

public class PalindromeArr {
      public static void palindrome(int arr[]) {
            int f = 0, s = arr.length - 1;

            while (f < s) {
                  if (arr[f] != arr[s]) {
                        System.out.println("The array is not palindrome");
                        return;
                  }
                  f++;
                  s--;
            }
            System.out.println("The array is palindrome");
      }

      public static void main(String args[]) {
            int[] nums = { 1, 2, 3,4, 2, 1 };
            palindrome(nums);
      }
}

File: cognizent_GENCNEXT/PalindromeArr.cs
// program to check array is palindrome or not
using System;

namespace cognizent_GENCNEXT
{
    public class PalindromeArr
    {
        public static void palindrome(int[] arr)
        {
            int f = 0, s = arr.Length - 1;

            while (f < s)
            {
                if (arr[f] != arr[s])
                {
                    Console.WriteLine("The array is not palindrome");
                    return;
                }
                f++;
                s--;
            }
            Console.WriteLine("The array is palindrome");
        }

        public static void Main(string[] args)
        {
            int[] nums = { 1, 2, 3, 4, 2, 1 };
            palindrome(nums);
        }
    }
}
